### PR TITLE
build-style/cmake: add check fo nodebug flag

### PR DESCRIPTION
**Pull request by Johnnynator**

### CODE REVIEW

File: srcpkgs/qt5-webkit/template [hunk 1]
```diff
@@ -36,7 +36,7 @@ case "$XBPS_TARGET_MACHINE" in
 		;;
 	i686*) 	# try to reduce memory footprint when linking by using gold
 		nodebug=1
-		configure_args+=" -DENABLE_ALLINONE_BUILD=OFF"
+		configure_args+=" -DUSE_LD_GOLD=ON"
```
Comment (Gottox) [inline srcpkgs/qt5-webkit/template]: is this on purpose?

Comment (Johnnynator) [reply srcpkgs/qt5-webkit/template]: I didn't clean up my branch before pushing. Will clean it once the local build finished.

Comment (Gottox) [reply srcpkgs/qt5-webkit/template]: :+1:

**Comment by q66** [comment]
:+1:

**Comment by Hoshpak** [comment]
:shipit:

**Pull request merged**